### DISCUSSION TITLE
Server IP address change

### DISCUSSION THREAD

**Discussion by molarszbt18** [AdguardTeam/AdGuardHome#4191] [Q&A] [answered]
Had to change the IP of the server running AdGuardHome as it needed to move to a new VLAN.  OS IP change went flawless and removed old IP address.  Doing so killed Ad Guard.  I do not see a way to edit a file but looks like, per the log, AdGuard is trying to bind to the old IP address and not the new one.   I tried to uninstall and reinstall, and nothing.  Any ideas on how to fix this?

2022/01/24 10:21:01 [info] service: control action: run
2022/01/24 10:21:01.129617 [info] AdGuard Home, version v0.107.2
2022/01/24 10:21:01.129642 [info] AdGuard Home is running as a service
2022/01/24 10:21:01.129656 [info] This is the first time AdGuard Home is launched
2022/01/24 10:21:01.129663 [info] Checking if AdGuard Home has necessary permissions
2022/01/24 10:21:01.129915 [info] AdGuard Home can bind to port 53
2022/01/24 10:21:01.171555 [info] Initializing auth module: /opt/AdGuardHome/data/sessions.db
2022/01/24 10:21:01.191145 [info] auth: initialized.  users:0  sessions:0
2022/01/24 10:21:01.191203 [info] Initialize web module
2022/01/24 10:21:01.191235 [info] This is the first launch of AdGuard Home, redirecting everything to /install.html
2022/01/24 10:21:01.191314 [info] AdGuard Home is available at the following addresses:
2022/01/24 10:21:01.191642 [info] Go to http://127.0.0.1:3000
2022/01/24 10:21:01.191658 [info] Go to http://[::1]:3000
2022/01/24 10:21:01.191665 [info] Go to http://192.168.2.2:3000
2022/01/24 10:20:12.468984 [info] Received signal "terminated"
2022/01/24 10:20:12.469082 [info] stopping AdGuard Home
2022/01/24 10:20:12.469099 [info] stopping http server...
2022/01/24 10:20:12.471740 [info] stopped http server
2022/01/24 10:20:12.473581 [info] Stopped
fo] stopped http server
2022/01/24 10:11:57.601096 [info] Stopped
nnot assign requested address
the tcp listener loop on 192.168.1.2:53
2022/01/24 08:30:33.758006 [info] Received signal "terminated"
2022/01/24 08:30:33.758086 [info] stopping AdGuard Home
2022/01/24 08:30:33.758103 [info] stopping http server...
2022/01/24 08:30:33.759399 [info] stopped http server
2022/01/24 08:30:33.759479 [info] Stopping the DNS proxy server
2022/01/24 08:30:33.759606 [info] Stopped the DNS proxy server
"AdGuardHome.err" 12146L, 1576833B

**Comment by ainar-g** [comment, marked as answer]
Hello.  What do you mean by “I do not see a way to edit a file”?  The bind address can be changed in `AdGuardHome.yaml`, but the service must be stopped before that to avoid it overwriting the new values with the old ones.

**Reply by molarszbt18** [reply]
Sorry, where is that file located?  I am on Unix.  And given it fails to start I should be good to change it now.

**Reply by ainar-g** [reply]
That depends on how you installed AdGuard Home.  If you used the script in our README, and by “Unix” you don't mean macOS, then the default working directory is `/opt/AdGuardHome`.

**Reply by molarszbt18** [reply]
I do not see it in that folder: (and yes, I used the script)
<img width="346" alt="image" src="https://user-images.githubusercontent.com/55773395/150820364-b7dc2194-3c11-4300-9b2d-2c72541d1c2e.png">

**Reply by ainar-g** [reply]
That's rather weird.  The file should appear once the initial installation is done on the web interface.  Are you sure that this is the same `AdGuardHome` binary as the one that your server is trying to run?

**Reply by molarszbt18** [reply]
Yes, odd.  I copied the file from my primary, made changes to reflect the secondary, and it's back on and running.  Note this was Fedora 33, and was installed a few months back.  Not sure if it was an issue then or still happens in regards to the no YAML file.

**Reply by ainar-g** [reply]
So, is the issue fixed now then?

**Reply by molarszbt18** [reply]
Yes, testing has proven a success.  Feel free to close this out.
